content/en/tracing/profiling/_index.md

```diff
@@ -119,14 +119,18 @@ The Datadog Profiler requires Python 2.7+. Memory profiling only works on Python
 
 | Environment variable                             | Type          | Description                                                                                      |
 | ------------------------------------------------ | ------------- | ------------------------------------------------------------------------------------------------ |
-| `DD_PROFILING_API_KEY`                           | String        | The [Datadog API key][1] to use when uploading profiles.                                         |
```

Comment:
This should be consistent with other profilers which have (for now) intentionally gone with DD_PROFILING_API_KEY. We can revisit that but it should be done holistically. Please hold off on merging until that convo happens.

For the record, canonical doc- https://docs.google.com/spreadsheets/d/1qrz2msAPWcCD8X6zkaEHZ9ut80Uuu3x1aNexV33_cG0/edit